docs/content/users/quickstart.md

```diff
@@ -321,8 +321,6 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
         ddev config --project-type=laravel --docroot=public --create-docroot
```

Comment:
This workflow results in an error:

```shell
ddev start
Starting my-laravel-app...
Network ddev-my-laravel-app_default  Created
Container ddev-my-laravel-app-dba  Started
Container ddev-my-laravel-app-web  Started
Container ddev-my-laravel-app-db  Started
Container ddev-router  Running
Failed to start my-laravel-app: open /home/user13/code/playground/my-laravel-app/.env.example: no such file or directory
```

`.env.example` is created by the `ddev composer create --prefer-dist laravel/laravel`

Reply:
Ah, and our `ddev start` is happening earlier, thanks. I must have tested with an existing setup.

Reply:
The clone instructions are fine.

Reply:
I made it accept if the .env.example isn't there and just skip the whole thing. Great catch.